1.0/docs/devguide/properties.md

```diff
@@ -615,14 +615,44 @@ Example:
 
 ## Property change notification events (notify) {#notify}
 
-When a property is set to `notify: true`, an event,
-<code><var>property-name</var>-changed</code>, is fired whenever the property
-value changes. These events are used by the two-way data binding system, and can
-also notify external scripts and frameworks to respond to changes in the element.
-
-For more on property change notifications and data binding, see  [Property
-change notification and two-way binding](data-binding.html#property-notification).
-
+When a property is set to `notify: true`, an event is fired whenever the 
+property value changes. The event syntax is:
+
+    property-name-changed
```

Comment:
We should use `<code><var>property-name</var>-changed</code>` here (and `<code><var>property-name</var></code>` below) for consistency.